### PR TITLE
chore(lint): add autolinting

### PR DESCRIPTION
**Pull request by Todmy**

### CODE REVIEW

File: src/requestDeduplicator.ts [hunk 1]
```diff
@@ -17,5 +17,6 @@ export default async function serve(id, action, args) {
     ongoingRequests.set(key, requestPromise);
   }
 
-  return ongoingRequests.get(key);
+  return;
+  ongoingRequests.get(key);
```
Comment (wa0x6e) [inline src/requestDeduplicator.ts:21]: this is weird

**Review by wa0x6e — CHANGES_REQUESTED** [review]

File: src/requestDeduplicator.ts (hunk 1 above)
Comment (Todmy) [reply src/requestDeduplicator.ts:21]: omg 😆 . The lint fixed it in the wrong way

**Review by wa0x6e — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -29,14 +30,22 @@
     "ts-node": "^10.9.1",
     "typescript": "^4.1.3"
   },
+  "lint-staged": {
+    "*.{js,ts,json}": [
+      "eslint --fix",
```
[the remark this answers is not in the document]
Comment (Todmy) [reply package.json:35]: makes sense

File: package.json [hunk 3]
```diff
@@ -29,14 +30,22 @@
     "ts-node": "^10.9.1",
     "typescript": "^4.1.3"
   },
+  "lint-staged": {
+    "*.{js,ts,json}": [
+      "yarn lint:fix",
+      "git add"
```
Comment (wa0x6e) [inline package.json:36]: From my understanding of this package, we don't need this command, as it's the job of lint-staged

**Review by wa0x6e — CHANGES_REQUESTED** [review]

**Review by wa0x6e — APPROVED** [review]
utAck

**Pull request merged**